### DISCUSSION TITLE
关于xbox的设置问题，望各位大神指点一二，感激不尽

### DISCUSSION THREAD

**Discussion by trombonist852** [vernesong/OpenClash#3103] [Q&A]
请教下各位大神 openclash打开之后，win10 xbox网络里面性能延迟-1毫秒，数据包丢失100%，xboxlive多人游戏nat 类型开放，服务器连接已阻止，关掉openclash就正常了，这个怎么破？openclash版本为最新的103beta，模式为Redir-Host  TUN模式，规则附加和规则设置都添加了相关信息，但是就联不通，望知道的大神指点一二  感激不尽！
![微信图片_20230315110220](https://user-images.githubusercontent.com/13253090/225194998-824c5ce2-cdc7-4ee8-85f1-7a82d5d1e475.png)
![微信图片_20230315110731](https://user-images.githubusercontent.com/13253090/225195537-8a8ff82b-2f63-4f10-a38b-35f075ce98f8.png)
![微信图片_20230315110810](https://user-images.githubusercontent.com/13253090/225195555-b86d8a46-8141-463d-b6a0-1fa99d7e2f40.png)
![微信图片_20230315111618](https://user-images.githubusercontent.com/13253090/225196292-d61c7131-8126-4307-a291-b01aa4c21035.png)

**Comment by trombonist852** [comment]
有没有好心人能帮忙指点一下呀

**Comment by cpxn73** [comment]
同问，我的丢包率也是100%

**Reply by jelly21fish** [reply]
试试这个方法，先解析x1ds.xboxlive.com的ip，再让openclash绕过他：https://github.com/vernesong/OpenClash/issues/3123#issuecomment-2104784260
